### DISCUSSION TITLE
Smart Blinds Enhanced support

### DISCUSSION THREAD

**Discussion by Hedrakk** [milo526/homebridge-tuya-web#218] [Q&A]
Hi,

I've recently installed a Tuya Smart Blinds in my office room and I was able to set it up with Tuya Web pluggin with HOOBS. The issue I have is that in the Tuya App on my iPhone, I can set the Blinds Opening to any percentage but in either HOOBS or directly in Apple Home app, I can only Fully open or Fully close the blinds, I cannot set it to, let say 50% or 75%.

Maybe my Smart Blinds device is not appearing in HOOBS as the correct device Type or this ability to is not implemented yet. If I should already be able to control the opening percentage with the latest version of the Tuya Web pluggin, can you help me out on this?

Thanks

**Comment by milo526** [comment]
Percentage based opening is in the works see #148 
Using other apps (such as the eve home app) you should be able to use the "stop" functionality to partially open them.
